src/Faithlife.Utility/JetBrainsAnnotations.cs

```diff
@@ -0,0 +1,1237 @@
+// <auto-generated>From ReSharper » Options » Code Inspection » Code Annotations</auto-generated>
```

Comment:
Would it be worth naming this file `JetBrainsAnnotations.g.cs`?

(Are there some tools that are sensitive to that filename pattern and don't "inspect" it, etc.? I can't remember.)

Reply:
Good idea. Done.